cylc/flow/data_store_mgr.py

```diff
@@ -1658,13 +1733,9 @@ def insert_job(
         )
         # Not all fields are populated with some submit-failures,
         # so use task cfg as base.
-        j_cfg = pdeepcopy(self._apply_broadcasts_to_runtime(
-            tp_tokens,
-            self.schd.config.cfg['runtime'][tproxy.name]
-        ))
-        for key, val in job_conf.items():
-            j_cfg[key] = val
-        j_buf.runtime.CopyFrom(runtime_from_config(j_cfg))
+        j_buf.runtime.CopyFrom(
+            runtime_from_partial(job_conf, tproxy.runtime)
```

Comment:
Used instead, because it can pick up existing orphan runtime, or runtime of newly created post reload.